spec/serializers/organization_serializer_spec.rb

```diff
@@ -88,4 +88,22 @@
       end
     end
   end
+
+  describe "#urls" do
+    let(:result) do
+      OrganizationSerializer.single({}, Organization.where(id: organization.id), {})
```

Comment:
I can't find the docs for this method in ruby/rails/AR/ActiveModelSerializers. It makes sense but I'd like to know what the empty hashes signify, can you link that for me?

Reply:
Ah hah, a restpack method. That explains it, thanks.